plugins/generator-1.21.1/neoforge-1.21.1/templates/elementinits/menus.java.ftl

```diff
@@ -45,6 +45,46 @@ public class ${JavaModName}Menus {
 		= REGISTRY.register("${gui.getModElement().getRegistryName()}", () -> IMenuTypeExtension.create(${gui.getModElement().getName()}Menu::new));
 	</#list>
 
+	public interface MenuAccessor {
+	    HashMap<String, Object> getMenuState();
+	}
+
+	public static void updateMenuState(Player entity, int elementType, String name, Object elementState) {
+	    if (entity.containerMenu instanceof MenuAccessor menu) {
+        	HashMap<String, Object> menuState = menu.getMenuState();
+            if (elementType == 0) {

```

Comment:
Instead of ifs, you can just inline elementType, such as

`return (T) menuState.getOrDefault(elementType + ":" + name, defaultValue);`

Reply:
@151chit, this was not resolved, there are still ifs:

![image](https://github.com/user-attachments/assets/d2e153c7-7a3e-400c-9afe-2ddf90ca220e)

Indentation also seems a bit strange.

Applies to both MC versions